tests/e2e/scenarios/upgrade-ab/run-test.sh

```diff
@@ -88,7 +88,7 @@ ${KUBETEST2} \
     --kubernetes-version="${K8S_VERSION_A}" \
     --control-plane-size="${KOPS_CONTROL_PLANE_SIZE:-1}" \
     --template-path="${KOPS_TEMPLATE:-}" \
-    --create-args="--networking calico ${KOPS_EXTRA_FLAGS:-}"
```

Comment:
This wasn't intentional, I assume. We at least need to add this back so the scenario actually runs Karpenter.

Reply:
I added it back in https://github.com/kubernetes/kops/pull/15144/commits/684809d452a8741c74d30c0a821b3fbb2d56f823.
Is it really necessary as we drop the necessity of the Karpenter flag in https://github.com/kubernetes/kops/pull/15144/commits/1d457c438e77dd356de15b8db70ca46154bf2e97?

Reply:
Well. That is the flag that tells kOps to use karpenter-managed IGs instead of ASG-managed ones. When that one is dropped, kOps won't provision karpenter.